packages/block-editor/src/components/block-list/block-list-block-context.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
+
+export const BlockListBlockContext = createContext( null );
```

Comment:
Pulling this out to its own file so that it's available for use on both web and mobile (React native). Hopefully that fixes mobile tests.